README.md

```diff
@@ -14,6 +14,7 @@ Please open a Pull Request to add a new public contribution.
 * February 06, 2022 [Building a career in frontend development](https://docs.google.com/presentation/d/1e8naMPhJghbhDdE2VDBsk2JhQ4Etb1-amAPH2aY0kJk/edit?usp=sharing) - [Smile Gupta](https://twitter.com/smileguptaaa)
 
 ## 2021
+* Dec 16,2021 [AI powered Smart routing for payments(1:13:08)](https://drive.google.com/file/d/1bMfunAH62wuXrKeLSo2X6CzifFXicdh-/view) - [Aayush Gupta](https://twitter.com/aayushguptaa)
```

Comment:
let's give github?

Reply:
i only have the recording link. should i upload the file to github and give github link?

Reply:
updated